examples/machine-learning/a3-ultragpu-8g/a3ultra-vm.yaml

```diff
@@ -21,6 +21,8 @@ vars:
   deployment_name: a3ultra-vm-instance
   region: europe-west1
   zone: europe-west1-b
+  a3u_reservation_name: # supply reservation name
+  a3u_provisioning_model:  # supply provisioning model
```

Comment:
I suggest we supply the customer with RESERVATION_BOUND as the value here for now, especially as cloud.google.com documentation is limited. I believe this will be the typical use case.